x/recovery/spec/01_concepts.md

```diff
@@ -0,0 +1,92 @@
+# Concepts
+
+## **Key generation**
+
+`secp256k1` refers to the parameters of the elliptic curve used in generating cryptographic public keys. Like Bitcoin, IBC compatible chains like the Cosmos chain use `secp256k1` for public key generation.
+
+Some chains use different elliptic curves for generating public keys. An example is the`eth_secp256k1`used by Ethereum and Evmos chain for generating public keys.
+
+```
+// Generate new random ethsecp256k1 private key and address
+
+ethPrivKey, err := ethsecp256k1.GenerateKey()
+ethsecpAddr := sdk.AccAddress(ethPrivKey.PubKey().Address())
+
+// Bech32 "evmos" address
+ethsecpAddrEvmos := sdk.AccAddress(ethPk.PubKey().Address()).String()
+
+// We can also change the HRP to use "cosmos"
+ethsecpAddrCosmos := sdk.MustBech32ifyAddressBytes(sdk.Bech32MainPrefix, ethsecpAddr)
+```
+
+The above example code demonstrates a simple user account creation on Evmos. 
```

Comment:
you can fix the lint errors by using [markdownlint](https://github.com/DavidAnson/markdownlint#install):

```
markdownlint -f ./x/recovery/
```